modules/vsphere-anti-affinity.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/installing-restricted-networks-vsphere.adoc
+// * installing/installing_vsphere/installing-vsphere-network-customizations.adoc
+// * installing/installing_vsphere/installing-vsphere.adoc
+
+:_content-type: PROCEDURE
+[id="anti-affinity-vsphere_{context}"]
+= Configure vSphere DRS anti-affinity rules for control plane nodes
+
+vSphere Distributed Resource Scheduler (DRS) anti-affinity rules can be configured to support higher availability of OpenShift Control Plane nodes so that the vSphere Virtual Machines for these nodes are not scheduled to the same vSphere Host. 
+
+[IMPORTANT]
+====
+* The following information applies to compute DRS only and does not apply to storage DRS.
+
+* The `govc` command is an open-source command available from VMware; it is not available from Red Hat. `govc` is not supported by the Red Hat Enterprise Linux technical support organization. 
+
+* Instructions for downloading and installing 'govc' are found on the VMware documentation website.
+==== 
+
+// https://docs.vmware.com/en/VMware-Telco-Cloud-Operations/1.4.0/deployment-guide-140/GUID-5249E662-D792-4A1A-93E6-CF331552364C.html#:~:text=Govc%20is%20an%20open%20source,operations%20on%20the%20target%20vCenter.
+
+.Example command
+
+[source,terminal]
+----
+# govc cluster.rule.create \
+  -name openshift4-control-plane-group \
+  -dc MyDatacenter -cluster MyCluster \
+  -enable \
+  -anti-affinity master-0 master-1 master-2
+----
+
+After creating the rule, your control plane nodes will be automatically migrated by vSphere so they are not running on the same hosts. This might take some time while vSphere reconciles the new rule. Successful command completion is shown below:
+
+.Example output
+[source,terminal]
+----
+ [13-10-22 09:14:41] Reconfigure /MyDatacenter/host/MyCluster...OK
+----
+
+[NOTE]
+====
+The migration will occur automatically and might cause brief OpenShift API outage or latency until the migration is completed by vSphere.
+====
+
+== Maintaining vSphere DRS anti-affinity rules
+
+The vSphere DRS Anti-Affinity rules will need to be updated manually in the event of a control plane VM name change or migration to a new vSphere Cluster.
+
+.Procedure
+
+. Remove any existing DRS anti-affinity rule:
++
+[source,terminal]
+----
+# govc cluster.rule.remove \
```

Comment:
Does this really have to be run as root? If not, it should be "$". If yes, then you should explicit say that in a prepreq or a step. See the SSG on this topic: https://redhat-documentation.github.io/supplementary-style-guide/#commands-with-root-privileges

(several places throughout the PR, if an update is required)

Reply:
@bergerhoffer Unfortunately, yes, it has to be run as root. It's a vmware utility that influences control at the hypervisor level.  https://docs.vmware.com/en/VMware-Telco-Cloud-Operations/1.4.0/deployment-guide-140/GUID-5249E662-D792-4A1A-93E6-CF331552364C.html  Yes it gives me indigestion but QE confirmed the requirement also. :-/

Reply:
@GroceryBoyJr That's totally fine if it does need to be run as root! The guidance I linked is just saying that instead of only relying on the `#` prompt to indicate to run as a root user, you should also explicitly say that it is required in the step or in the prereqs.

I see that you change the `#` to `$`, and I don't see any text saying that root is required. Does that prompt need to be changed back, if root is really required?

(If you do have to change it back and add a sentence about being root, there is a double backtick in your PR now that could also be fixed)